### PR TITLE
Food Details Screen

### PR DESCRIPTION
**Pull request by aahenry51**

### CODE REVIEW

**Comment by aahenry51** [comment]
image

File: mobile/src/screens/foodDetails/FoodDetails.style.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    marginTop: size.rg,
+  },
+
+  button: {
+    padding: size.rg,
+    borderRadius: size.xxl,
+    marginHorizontal: size.xl,
+    marginTop: size.xl,
+  },
+  text: {
```
Comment (lasharela) [inline mobile/src/screens/foodDetails/FoodDetails.style.ts:61]: style name might be descriptive

File: mobile/src/screens/foodDetails/FoodDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  return (
+    <FoodDetailsScreenView
+      title="Ground Beef Tacos"
+      description="Brown the beef better. Lean ground beef – I like to use 85% lean angus. Garlic – use fresh chopped. Spices – chili powder, cumin, onion powder."
+      subtitle="Choice of Add On"
+      ratings={4.5}
+      reviewCount={31}
+      countPlusMinus={count}
+      onPressMinus={() => {
```
Comment (lasharela) [inline mobile/src/screens/foodDetails/FoodDetails.tsx:16]: maybe this logic will be outside, not inline

File: mobile/src/screens/foodDetails/FoodDetails.view.tsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+            </TouchableOpacity>
+          </View>
+        </View>
+      </>
+    );
+  };
+
+  const ratingsAndReview = () => {
+    return (
+      <>
```
Comment (lasharela) [inline mobile/src/screens/foodDetails/FoodDetails.view.tsx:52]: remove

File: mobile/src/screens/foodDetails/FoodDetails.view.tsx [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+          </Text>
+        </View>
+      </>
+    );
+  };
+
+  const addOnSelections = (imageSource, title, price) => {
+    const [Selected, SelectedChange] = useState(false);
+    return (
+      <>
```
Comment (lasharela) [inline mobile/src/screens/foodDetails/FoodDetails.view.tsx:72]: remove

File: mobile/src/screens/foodDetails/FoodDetails.view.tsx [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+            <Text style={styles.titleAddOn}>{title}</Text>
+          </View>
+          <TouchableOpacity
+            style={styles.containerAddOnPriceSelection}
+            onPress={() => {
+              Selected ? SelectedChange(false) : SelectedChange(true);
+            }}>
+            <Text style={styles.titleAddOn}>{`+${price}`}</Text>
+
+            {Selected ? (
```
Comment (lasharela) [inline mobile/src/screens/foodDetails/FoodDetails.view.tsx:85]: why this is capital?

**Review by lasharela — APPROVED** [review]

File: mobile/src/screens/foodDetails/FoodDetails.view.tsx [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+  };
+
+  const ratingsAndReview = () => {
+    return (
+      <>
+        <View style={styles.containerRatingReview}>
+          <Icon name="Star" width={size.rg} height={size.rg} />
+          <Text style={styles.ratings}>{ratings}</Text>
+          {reviewCount > 30 ? (
+            <Text style={styles.reviewCount}>{`(30+)`}</Text>
```
Comment (raynesmike) [inline mobile/src/screens/foodDetails/FoodDetails.view.tsx:57]: maybe you can create a variable for the 30. like maxCount or someting. then use that with interpolation

**Review by raynesmike — APPROVED** [review]

**Comment by aahenry51** [comment]
image

**Review by mshago — APPROVED** [review]

**Review by Dulaya — APPROVED** [review]
Make sure you fix the conflicts.

**Review by leo-he-1 — APPROVED** [review]

**Pull request merged**